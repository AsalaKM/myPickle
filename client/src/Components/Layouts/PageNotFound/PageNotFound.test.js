import React from "react"
import ReactDOM from "react-dom"
import { Error404, Error500 } from "./index"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Error404 />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Error500 />, div)
  ReactDOM.unmountComponentAtNode(div)
})
