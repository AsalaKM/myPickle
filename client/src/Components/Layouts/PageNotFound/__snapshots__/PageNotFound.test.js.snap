// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 matches snapshot 1`] = `
<div
  className="sc-bwzfXH jwfMIB"
>
  <div
    className="sc-bdVaJa ccpsQN"
  >
    <div
      className="sc-bxivhb dKkUBg"
    >
      <svg
        className="error-logo"
      >
        pickle_logo.svg
      </svg>
      <h1
        className="sc-ifAKCX byHice"
      >
        4
        <span>
          0
        </span>
        4
      </h1>
      <p
        className="sc-EHOje hJIuCB"
      >
        Sorry, we can't find the page you are looking for
      </p>
      <a
        className="sc-bZQynM JyOBu"
        href="/"
        onClick={[Function]}
      >
        <h3>
          Return home
        </h3>
      </a>
    </div>
  </div>
</div>
`;

exports[`500 matches snapshot 1`] = `
<div
  className="sc-bwzfXH jwfMIB"
>
  <div
    className="sc-bdVaJa ccpsQN"
  >
    <div
      className="sc-bxivhb dKkUBg"
    >
      <svg
        className="error-logo"
      >
        pickle_logo.svg
      </svg>
      <h1
        className="sc-ifAKCX byHice"
      >
        Oops!! 
        <span>
          5
        </span>
        00
      </h1>
      <p
        className="sc-EHOje hJIuCB"
      >
        Sorry, looks like somthing went wrong!
      </p>
      <a
        className="sc-bZQynM JyOBu"
        href="/"
        onClick={[Function]}
      >
        <h3>
          Return home
        </h3>
      </a>
    </div>
  </div>
</div>
`;
